peachjam/views/authors.py

```diff
@@ -27,9 +27,10 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         doc_types = list(
             set(
-                self.form.filter_queryset(
-                    self.get_base_queryset(), exclude="doc_type"
-                ).values_list("doc_type", flat=True)
+                self.form.filter_queryset(self.get_base_queryset(), exclude="doc_type")
+                .order_by()
```

Comment:
this clears the ordering applied by the form, otherwise we'll fetch fields required for ordering, which we don't care about